src/stylesheets/mixins/media-breakpoint.scss

```diff
@@ -0,0 +1,5 @@
+@mixin media-breakpoint($size) {
```

Comment:
@rafayet-monon Why don't we use mixin from bootstrap?

Reply:
Fixed in [89e392b](https://github.com/rafayet-monon/surveyor/pull/9/commits/89e392b80eede7fd5a80ecacb83653f4d7d838be)